package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@nextcloud/axios": "^1.3.1",
     "escape-html": "^1.0.3",
+    "moment": "^2.24.0",
```

Comment:
```suggestion
    "@nextcloud/moment": "^1.1.0",
```

Reply:
That has the advantage that the user locale is actually set to *our* value :)

Reply:
yep let me change that